packages/instant/src/components/asset_amount_input.tsx

```diff
@@ -7,26 +7,51 @@ import { ERC20Asset } from '../types';
 import { assetUtils } from '../util/asset';
 import { util } from '../util/util';
 
-import { AmountInput, AmountInputProps } from './amount_input';
+import { ScalingAmountInput } from './scaling_amount_input';
 import { Container, Text } from './ui';
 
 // Asset amounts only apply to ERC20 assets
-export interface AssetAmountInputProps extends AmountInputProps {
+export interface AssetAmountInputProps {
     asset?: ERC20Asset;
     onChange: (value?: BigNumber, asset?: ERC20Asset) => void;
+    startingFontSizePx: number;
+    fontColor?: ColorOption;
 }
 
-export class AssetAmountInput extends React.Component<AssetAmountInputProps> {
+export interface AssetAmountInputState {
+    currentFontSizePx: number;
+}
+
+export class AssetAmountInput extends React.Component<AssetAmountInputProps, AssetAmountInputState> {
     public static defaultProps = {
         onChange: util.boundNoop,
     };
+    constructor(props: AssetAmountInputProps) {
+        super(props);
+        this.state = {
+            currentFontSizePx: props.startingFontSizePx,
+        };
+    }
     public render(): React.ReactNode {
         const { asset, onChange, ...rest } = this.props;
         return (
             <Container>
-                <AmountInput {...rest} onChange={this._handleChange} />
+                <Container borderBottom="1px solid rgba(255,255,255,0.3)" display="inline-block">
```

Comment:
Should we be using a `ColorOption` instead of hard-coding `rgba(255,255,255,0.3)` here?

Reply:
I've resigned to just use a string in this case (and a few others that are opacity-based) because "white with opacity 0.3" is not really a color.

Reply:
Maybe we can add a constant somewhere?

Reply:
hypothetically we could do something with this https://github.com/misund/hex-to-rgba#readme but def seems like overkill.

I like the idea of all of our colors being in one file, or at least easily findable.  I'm open to constants, or maybe even a `whiteWithOpacity` function in theme.ts, or something like that: open to diff implementations but overall would prefer to have the colors centralized, and not just one-off defined components

Reply:
@BMillman19 you can't really go from rgba to HEX, unless I'm misunderstanding what you mean.

Reply:
I'm just going to make this a constant in `theme.ts`

To get access to the actual injected theme you need to read from context, define a styled-component or use a HOC.